lib/holdings/mhld.rb

```diff
@@ -48,7 +48,9 @@ def standard_or_zombie_library
     end
 
     def sanitize_mhld_data(data)
-      CGI.escape_html(data).gsub('),', '), ').gsub(/,\s?/, ', ').gsub('-', '-<wbr/>').html_safe if data.present?
+      return if data.blank?
+
+      CGI.escape_html(data.gsub(/;\s?/, '; ').gsub('),', '), ').gsub(/,\s?/, ', ')).gsub('-', '-<wbr/>').html_safe
```

Comment:
Yes, some of the punctuation changes are now operating on the raw data before it is html escaped.

Semicolon escaping needed to happen there because encoded entities like `&gt;`will have semicolons in their escaped form that don't need the extra whitespace..